src/modules/assertions/require_inline_coding.ts

```diff
@@ -0,0 +1,12 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+
+export const  requireInlineCoding = () => {
+  const sol_file_Regex = /http:\/\/\S+.sol/g;
+  if (sol_file_Regex.test(JSON.stringify(context.payload?.pull_request?.body))) {
+    console.log(`EIP-BOT is terminating. Please replace external link http://.../file.sol for inline code reference..`);
```

Comment:
We should show the matched string in the error.  Something like `sol_file_Regex.match(...)` and then check if there are any matches, and if are include them in the log message.

Reply:
done